mp/src/game/shared/momentum/weapon/weapon_mom_stickybomblauncher.cpp

```diff
@@ -19,6 +19,8 @@
 #define MOM_STICKYBOMB_MAX_CHARGE_VEL 2400
 #define MOM_STICKYBOMB_MAX_CHARGE_TIME 4.0f
 
+#define MOM_STICKYLAUNCHER_BUFFER_WINDOW 0.2f
```

Comment:
Should this be MOM_STICKYLAUNCHER_DET_BUFFER_WINDOW instead to be more specific?